azuredevops/boards.go

```diff
@@ -8,11 +8,11 @@ import (
 // BoardsService handles communication with the boards methods on the API
 // utilising https://docs.microsoft.com/en-gb/rest/api/vsts/work/boards
 type BoardsService struct {
-	client *Client
+	client *ProjectClient
 }
 
-// ListBoardsResponse describes the boards response
-type ListBoardsResponse struct {
+// BoardsListResponse describes the boards response
+type BoardsListResponse struct {
```

Comment:
Just out of interest, is this a rename because it's in the Boards service? Curious.

Reply:
I did this in order to be more consistent with other ListResponse types - BuildDefinitionsListResponse, BuildsListResponse, DeliveryPlansListResponse, etc.